scale/product/views.py

```diff
@@ -31,20 +31,28 @@ def list(self, request):
         started = rest_util.parse_timestamp(request, 'started', required=False)
         ended = rest_util.parse_timestamp(request, 'ended', required=False)
         rest_util.check_time_range(started, ended)
+        time_field = rest_util.parse_string(request, 'time_field', required=False,
+                                            accepted_values=ProductFile.VALID_TIME_FIELDS)
 
         job_type_ids = rest_util.parse_int_list(request, 'job_type_id', required=False)
         job_type_names = rest_util.parse_string_list(request, 'job_type_name', required=False)
         job_type_categories = rest_util.parse_string_list(request, 'job_type_category', required=False)
         is_operational = rest_util.parse_bool(request, 'is_operational', required=False)
         is_published = rest_util.parse_bool(request, 'is_published', default_value=True)
         file_name = rest_util.parse_string(request, 'file_name', required=False)
+        job_output = rest_util.parse_string(request, 'job_output', required=False)
+        recipe_ids = rest_util.parse_int_list(request, 'recipe_id', required=False)
+        recipe_type = rest_util.parse_string(request, 'recipe_type', required=False)
```

Comment:
Shouldn't *recipe_type* be *recipe_type_ids* and an integer list instead of a string? That would match the REST API docs and how *job_type_ids* was done.